azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -1,631 +1,657 @@
-        :param str mapping_name: Pre-defined mapping of the table. Required when stream_format is json/avro.
-        """
-        stream_format = stream_format.value if isinstance(stream_format, DataFormat) else DataFormat(stream_format.lower()).value
-        endpoint = self._streaming_ingest_endpoint + database + "/" + table + "?streamFormat=" + stream_format
-        if mapping_name is not None:
-            endpoint = endpoint + "&mappingName=" + mapping_name
-
-        self._execute(endpoint, database, None, stream, KustoClient._streaming_ingest_default_timeout, properties)
-
-    def _execute(self, endpoint: str, database: str, query: str, payload: io.IOBase, timeout: timedelta, properties: ClientRequestProperties = None):
-        """Executes given query against this client"""
-        request_headers = copy(self._request_headers)
-        json_payload = None
-        if not payload:
-            json_payload = {"db": database, "csl": query}
-            if properties:
-                json_payload["properties"] = properties.to_json()
-
-            client_request_id_prefix = "KPC.execute;"
-            request_headers["Content-Type"] = "application/json; charset=utf-8"
-        else:
-            if properties:
-                request_headers.update(json.loads(properties.to_json())["Options"])
-
-            client_request_id_prefix = "KPC.execute_streaming_ingest;"
-            request_headers["Content-Encoding"] = "gzip"
-
-        request_headers["x-ms-client-request-id"] = client_request_id_prefix + str(uuid.uuid4())
-
-        if properties is not None:
-            if properties.client_request_id is not None:
-                request_headers["x-ms-client-request-id"] = properties.client_request_id
-            if properties.application is not None:
-                request_headers["x-ms-app"] = properties.application
-            if properties.user is not None:
-                request_headers["x-ms-user"] = properties.user
-
-        if self._auth_provider:
-            request_headers["Authorization"] = self._auth_provider.acquire_authorization_header()
-
-        if properties:
-            timeout = properties.get_option(ClientRequestProperties.request_timeout_option_name, timeout)
-
-        response = self._session.post(endpoint, headers=request_headers, data=payload, json=json_payload, timeout=timeout.seconds)
-
-        if response.status_code == 200:
-            if endpoint.endswith("v2/rest/query"):
-                return KustoResponseDataSetV2(response.json())
-            return KustoResponseDataSetV1(response.json())
-
-        if payload:
-            raise KustoServiceError(
-                "An error occurred while trying to ingest: Status: {0.status_code}, Reason: {0.reason}, Text: {0.text}".format(response), response
-            )
-
-        raise KustoServiceError([response.json()], response)
+"""A module to make a Kusto request."""
+import io
```

Comment:
did you change line endings or something? the entire file shouldn't be changed

Reply:
Not sure what do you mean by this